src/Services/LightningService.cs

```diff
@@ -397,29 +394,20 @@ await foreach (var response in openStatusUpdateStream.ResponseStream.ReadAllAsyn
                                     };
 
                                     var channelfundingTx = fundedPSBT.GetGlobalTransaction();
+                                    var totalOut = new Money(channelOperationRequest.SatsAmount, MoneyUnit.Satoshi);
 
-                                    //We manually fix the change (it was wrong from the Base template due to nbitcoin requiring a change on a PSBT)
-
-                                    var totalIn = new Money(0L);
-
-                                    foreach (var input in fundedPSBT.Inputs)
+                                    if (!channelOperationRequest.Changeless)
                                     {
-                                        totalIn += (input.GetTxOut()?.Value);
+                                        //We merge changeFixedPSBT with the other PSBT with the change fixed
+                                        channelfundingTx.Outputs[0].Value -= totalOut + totalFees;
+                                        fundedPSBT = channelfundingTx.CreatePSBT(network).UpdateFrom(fundedPSBT);
                                     }
```

Comment:
I don't remember why I used totalIn for the first output, worth checking the numbersI don't remember why I used totalIn for the first output, worth checking the numbers

Reply:
Will leave it like it was before. Because actually the values differ for around 20 sats, which I don't understand very well why